kroxylicious-operator/src/main/resources/log4j2.yaml

```diff
@@ -27,7 +27,6 @@ Configuration:
         AppenderRef:
           - ref: STDOUT
       - name: io.javaoperatorsdk
-        level: INFO
```

Comment:
do we need this config block for `io.javaoperatorsdk` if we are removing the level override?